packages/utils/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@quality-metrics/utils",
+  "version": "0.0.1",
+  "dependencies": {
+    "bundle-require": "^4.0.1"
```

Comment:
Same issue as for `models` package, these aren't the correct dependencies.

Reply:
fixed